test/python/WMCore_t/WMRuntime_t/Scripts_t/SetupCMSSWPset_t.py

```diff
@@ -297,6 +298,15 @@ def _mixingModulesInputFilesTest(self, modules, pileupSubDict, seLocalName):
                  "filelist '%s' are not identical." % (filesInConfigDict, fileNames))
             self.assertEqual(sorted(filesInConfigDict), sorted(fileNames), m)
 
+    def testGetStdOutLogger(self):
```

Comment:
As I made further changes to this PR, this unit test is likely no longer required. Just a self reminder.